src/containers/permissionssummary/components/AllPermissions.js

```diff
@@ -68,26 +68,22 @@ class AllPermissions extends React.Component {
     const tables = [];
 
     propertyPermissions.keySeq().forEach((property) => {
-      if (propertyPermissions.hasIn([property, 'userPermissions'])
-          && propertyPermissions.hasIn([property, 'rolePermissions'])) {
-
-        const rolePermissions = propertyPermissions.getIn([property, 'rolePermissions'], Immutable.Map());
-        const userPermissions = propertyPermissions.getIn([property, 'userPermissions'], Immutable.List());
-
-        const header = <h3 key={`header-${property}`}>{property} Permissions</h3>;
-        const roleTable = (<RolePermissionsTable
-            rolePermissions={rolePermissions}
-            headers={R_HEADERS}
-            key={`role-${property}`} />);
-        const userTable = (<UserPermissionsTable
-            property={property}
-            userPermissions={userPermissions}
-            rolePermissions={rolePermissions}
-            headers={U_HEADERS}
-            key={`user-${property}`} />);
-
-        tables.push(header, roleTable, userTable);
-      }
+      const rolePermissions = propertyPermissions.getIn([property, 'rolePermissions']) || Immutable.Map();
```

Comment:
`getIn` takes an optional second parameter, `notSetValue`, which is returned if the get doesn't find anything. You can do this:

```
const rolePermissions = propertyPermissions.getIn([property, 'rolePermissions'], Immutable.Map());
```

Reply:
Fixed

Reply:
Can't believe I keep forgetting this ><